src/quo2/components/avatars/group_avatar/view.cljs

```diff
@@ -7,23 +7,23 @@
             [quo2.components.avatars.group-avatar.style :as style]))
 
 (def sizes
-  {:icon      {:x-small 12
-               :small   16
-               :medium  16
-               :large   20
-               :x-large 32}
-   :container {:x-small 20
-               :small   28
-               :medium  32
-               :large   48
-               :x-large 80}})
+  {:icon      {:size/xs-20 12
+               :size/sm-28 16
+               :size/md-32 16
+               :size/lg-48 20
+               :size/xl-80 32}
+   :container {:size/xs-20 20
+               :size/sm-28 28
+               :size/md-32 32
+               :size/lg-48 48
+               :size/xl-80 80}})
```

Comment:
@codemaster115 I'm a bit confused here, `:size/xs-20` is `12` for icons & `20` for containers, then why we are using `-20` in both keywords?

cc: @J-Son89

Reply:
-20 is referring to the size property marked in figma. Lots of components have this property to handle the different variants.

Reply:
> @codemaster115 I'm a bit confused here, :size/xs-20 is 12 for icons & 20 for containers, then why we are using -20 in both keywords?

Me too @ajayesivan.

@codemaster115 - We can merge the size map to be more cleaner.

```clojure
{:size/xs-20 {:container-size 20
              :icon-size      12}
{:size/sm-28 {:container-size 28
              :icon-size      16}
```

<br />
<hr />

Not a fan of using `xs`, `sm`, `lg`, `xl`, ...etc with size numbers.

From my knowledge, the lowest size for a component is `14` in the `preview-list` component. So, I would presumably assign `14` for `xxxs-14`. But, If there is any other component that has size `12`, the `xxxs` would mean that. 

Can we use a common key like `:size/size-20` or `:size/s-20`?

Reply:
Yep I agree to both points here @smohamedjavid!